packages/components/src/Message/Message.stories.mdx

```diff
@@ -25,6 +26,18 @@ import { Message } from '@marigold/components';
 
 <Preview>
   <Story name="Message">
-    <Message>Hello, I am a simple message.</Message>
+    <Message variant="warning" title="Danger Zone!">
```

Comment:
Info: if i remove title prop here, storybook shows just the icon and no error. I think its special storybook because in the test I get an error if I remove title prop.

Reply:
You're right I shouldn't overwrite title. I rename it to messageTitle